sources/packages/api/src/route-controllers/institution-locations/institution-location.aest.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller, Get, Param } from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { ClientTypeBaseRoute } from "../../types";
+import BaseController from "../BaseController";
+import { InstitutionLocationControllerService } from "./institution-location.controller.service";
+import { InstitutionLocationAPIOutDTO } from "./models/institution-location.dto";
+
+/**
+ * Institution location controller for institutions Client.
+ */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("institution/location")
+@ApiTags(`${ClientTypeBaseRoute.AEST}-institution`)
+export class InstitutionLocationAESTController extends BaseController {
+  constructor(
+    private readonly locationControllerService: InstitutionLocationControllerService,
+  ) {
+    super();
+  }
+
+  /**
+   * Controller method to get institution locations with designation status for the given institution.
+   * @param institutionId
+   * @returns An array of InstitutionLocationsDetailsDto.
```

Comment:
IMO opinion we should avoid this type of comment where the return contains basically the type of the object returned. This information can be easily noticed in the code or in the IntelliSense while consuming a method.
@ann-aot FYI